mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -0,0 +1,449 @@
+				{
+				  "device": "/dev/vda",
+				  "partitions": [
+					{
+					  "label": "bios-1",
+					  "sizeMiB": 1,
+					  "typeGuid": "21686148-6449-6E6F-744E-656564454649"
+					},
+					{
+					  "label": "esp-1",
+					  "sizeMiB": 127,
+					  "typeGuid": "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
+					},
+					{
+					  "label": "boot-1",
+					  "sizeMiB": 384
+					},
+					{
+					  "label": "root-1"
+					}
+				  ],
+				  "wipeTable": true
+				},
+				{
+				  "device": "/dev/vdb",
+				  "partitions": [
+					{
+					  "label": "bios-2",
+					  "sizeMiB": 1,
+					  "typeGuid": "21686148-6449-6E6F-744E-656564454649"
+					},
+					{
+					  "label": "esp-2",
+					  "sizeMiB": 127,
+					  "typeGuid": "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
+					},
+					{
+					  "label": "boot-2",
+					  "sizeMiB": 384
+					},
+					{
+					  "label": "root-2"
+					}
+				  ],
+				  "wipeTable": true
+				}
+			  ],
+			  "filesystems": [
+				{
+				  "device": "/dev/disk/by-partlabel/esp-1",
+				  "format": "vfat",
+				  "label": "esp1",
+				  "wipeFilesystem": true
+				},
+				{
+					"device": "/dev/disk/by-partlabel/esp-2",
+					"format": "vfat",
+					"label": "esp2",
+					"wipeFilesystem": true
+				},
+				{
+				  "device": "/dev/md/md-boot",
+				  "format": "ext4",
+				  "label": "boot",
+				  "wipeFilesystem": true
+				},
+				{
+				  "device": "/dev/mapper/root",
+				  "format": "xfs",
+				  "label": "root",
+				  "wipeFilesystem": true
+				}
+			  ],
+			  "luks": [
+				{
+				  "clevis": {
+					"tpm2": true
+				  },
+				  "device": "/dev/md/md-root",
+				  "label": "luks-root",
+				  "name": "root",
+				  "wipeVolume": true
+				}
+			  ],
+			  "raid": [
+				{
+				  "devices": [
+					"/dev/disk/by-partlabel/boot-1",
+					"/dev/disk/by-partlabel/boot-2"
+				  ],
+				  "level": "raid1",
+				  "name": "md-boot",
+				  "options": [
+					"--metadata=1.0"
+				  ]
+				},
+				{
+				  "devices": [
+					"/dev/disk/by-partlabel/root-1",
+					"/dev/disk/by-partlabel/root-2"
+				  ],
+				  "level": "raid1",
+				  "name": "md-root"
+				}
+			  ]
+			}
+		  }`)
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Run:         runBootMirrorTest,
+		ClusterSize: 0,
+		Name:        `coreos.boot-mirror`,
+		Platforms:   []string{"qemu-unpriv"},
+		// skipping this test on UEFI until https://github.com/coreos/coreos-assembler/issues/2039
+		// gets resolved.
+		ExcludeFirmwares: []string{"uefi"},
+		Tags:             []string{"boot-mirror", "raid1"},
+		FailFast:         true,
+	})
+	register.RegisterTest(&register.Test{
+		Run:         runBootMirrorLUKSTest,
+		ClusterSize: 0,
+		Name:        `coreos.boot-mirror.luks`,
+		Platforms:   []string{"qemu-unpriv"},
+		// aarch64 and ppc64le are added temporarily to avoid test failure
+		// until we support specifying FCC directly in kola.
+		ExcludeArchitectures: []string{"aarch64", "ppc64le", "s390x"}, // no TPM support for s390x in qemu
```

Comment:
TPM comment is a copypaste error.

Reply:
yeah, let me fix it, but the comment is still valid though